js/schema/plugin.json

```diff
@@ -86,6 +92,11 @@
         "template": {
           "description": "A template string to build up the snippet",
           "type": "string"
+        },
+        "type": {
+          "description": "Backend type that the snippet works with",
+          "type": "string",
+          "default": ""
```

Comment:
Should this also be an `enum`?

Reply:
yep i think so, i have validation on the backend